Bottom: 7d6613e44d11a71c864b6f1f8d9944aef9258953
Top:    2a2dc86b7f46bcec2da42086f5947f4378c33a14
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-04-24 10:40:46 -0400

Refresh of uart.patch

---

diff --git a/arch/arm/boot/dts/ea3131.dts b/arch/arm/boot/dts/ea3131.dts
index 7513231..e1f2e51 100644
--- a/arch/arm/boot/dts/ea3131.dts
+++ b/arch/arm/boot/dts/ea3131.dts
@@ -108,9 +108,8 @@
 			reg = <0x20020000 0x100 0x20030000 0x100>;
 			interrupt-parent = <&evtr>;
 			interrupts = <4 4>;
-			bits = <16>;
+			interface-width = <16>;
 			no-eeprom;
-			simple-phy;
 		};			
 	};
 	memory@30000000 {
diff --git a/drivers/net/ethernet/davicom/dm9000.c b/drivers/net/ethernet/davicom/dm9000.c
index 8f6dfe9..1476bd6 100644
--- a/drivers/net/ethernet/davicom/dm9000.c
+++ b/drivers/net/ethernet/davicom/dm9000.c
@@ -1369,7 +1369,7 @@ static void  __devinit dm9000_parse_options(struct device_node *node, struct boa
 	const unsigned int *prop;
 	int len, bits;
 
-	prop = of_get_property(node, "bits", &len);
+	prop = of_get_property(node, "interface-width", &len);
 	if (prop) {
 		bits = __be32_to_cpup(prop);
 		switch (bits) {
@@ -1384,11 +1384,13 @@ static void  __devinit dm9000_parse_options(struct device_node *node, struct boa
 			break;
 		}
 	}
-	prop = of_get_property(node, "no_eeprom", &len);
+	prop = of_get_property(node, "no-eeprom", &len);
 	if (prop)
 		db->flags |= DM9000_PLATF_NO_EEPROM;
-	prop = of_get_property(node, "simple_phy", &len);
+	prop = of_get_property(node, "phy", &len);
 	if (prop)
+		db->flags |= DM9000_PLATF_EXT_PHY;
+	else
 		db->flags |= DM9000_PLATF_SIMPLE_PHY;
 #endif
 }
